Example-Crypto-REXX/README.md

```diff
@@ -0,0 +1,105 @@
+## Example-Crypto-REXX
+
+This samples queries Image Activation profiles to list crypto attributes for LPARs. It stores the results, which are in .csv format, in a z/OS data set member.  You can import the content into Microsoft™ Excel or another application of your choice.
+
+This sample uses **HWIREST API** to do the following:
+- List CPCs and retrieve the URI and target name associated with the LOCAL CPC or the *CPCname* provided
+- List the LPARs on the designated CPC and query the LPAR status.
+- For each LPAR matching the reqested status (default is operating), retrieve the following information:
+     - crypto-activity-cpu-counter-authorization-control
+     - assigned-crypto-domains
+     - assigned-cryptos
+- Note the crypto attributes are available on z16 or higher processors only.
+
+## System Prep work
+- Store RXCRYPT1 into a data set
+- Ensure your z/OS user ID has at least READ access to the following FACILITY Class Profiles
+    - HWI.TARGET.netid.nau
+    - HWI.TARGET.netid.nau.imagename
+
+    <p>where netid.nau represents the 3– to 17– character SNA name of the particular CPC whose LPARs are being queried and imagename represents the 1– to 8- character LPAR name, or * to represent all of the LPARs available on that CPC </p>
+
+- Allocate a partitioned data set that can hold the generated content
+   - the longest line generated depends on how many assigned-crypto-domains and assigned-cryptos are found
+   - minimum data set characteristics
+     - RECFM: VB
+     - LRECL: 800
+
+## Invocation
+**Syntax**:
+```
+  RXCRYPT1 -D outputDataSet [-C CPCname] [-S Status] [-I] [-V]                                                 
+ ```
+ where:
+  - *-D outputDataSet*
+      - **required**
+      - *outputDataSet* is the name of an existing partitioned data set
+      - if the query is successful, a member containing the crypto
+        information in a CSV format will be stored into the specified data set: *outputDataSet(memberName)*
```

Comment:
fix indentation please for the outputDataSet entity

Reply:
fixed